src/compass/utils/age.py

```diff
@@ -126,18 +126,21 @@ def run(cslc_file, cr_file, csv_output_file=None, plot_age=False,
 
     x_peak_vect = []
     y_peak_vect = []
+    snr_peak_vect = []
 
     # Find peak location for every corner reflector in DataFrame
     for idx, row in cr_df.iterrows():
-        x_peak, y_peak = find_peak(cslc_file, int(row['CR_X_CSLC']),
+        x_peak, y_peak, snr_peak = find_peak(cslc_file, int(row['CR_X_CSLC']),
                                    int(row['CR_Y_CSLC']), pol=pol,
                                    mission_id=mission_id, ovs_factor=ovs_factor,
                                    margin=margin)
         x_peak_vect.append(x_peak)
         y_peak_vect.append(y_peak)
+        snr_peak_vect.append(snr_peak)
 
     cr_df['CR_X_CSLC_PEAK'] = x_peak_vect
     cr_df['CR_Y_CSLC_PEAK'] = y_peak_vect
+    cr_df['CR_SNR_PEAK'] = snr_peak_vect
```

Comment:
That might be me but I would call this variable only as "CR_SNR". Adding peak is confusing as this metric is not representative of the peak but is a quality metric for the CR

Reply:
Makes sense. I'll change the variable names accordingly.